scss/utilities/rounded-corners-mixin.scss

```diff
@@ -11,13 +11,28 @@
   border-image-source: url('data:image/svg+xml;utf8,<?xml version="1.0" encoding="UTF-8" ?><svg version="1.1" width="5" height="5" xmlns="http://www.w3.org/2000/svg"><path d="M2 1 h1 v1 h-1 z M1 2 h1 v1 h-1 z M3 2 h1 v1 h-1 z M2 3 h1 v1 h-1 z" fill="rgb(#{red($color)},#{green($color)},#{blue($color)})" /></svg>');
 }
 
+@mixin border-image-chrome() {
+  @media all and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: .001dpcm) {
+    border-image-repeat: space;
+  }
+}
+
+@mixin border-image-ff() {
+  @-moz-document url-prefix() {
+    border-image-repeat: stretch;
```

Comment:
`border-image-repeat: stretch;` has already been specified in another place.

* L31
* L52

Reply:
Yes, but the thing is the Chrome mixin (L14-L18) is overriding the `border-image-repeat` to `space` in Firefox! 

So my thought process was:

1. Set the `border-image-repeat` to`stretch` to all of the browsers;
2. If the browser is Chrome, set the `border-image-repeat` to `space`;
3. If the browser is Firefox, set the `border-image-repeat` back to `stretch`, because as I said before, it's being overriding by the Chrome mixin.

I could remove L31 and L52, but I can't test in all of the browsers out there, to see if they're going to have a `border-image-repeat` property included by any of those two mixins. So In my opinion I would keep this as it is, or refactor Chrome mixin to a CSS code that only works in Chrome and nowhere else.

See the example below in Firefox (Storybook) to understand better what I'm saying about overriding mixins:

![image](https://user-images.githubusercontent.com/22016005/53742901-70254880-3e78-11e9-9bb8-28418d7f3e6b.png)

What do you think?